### DISCUSSION TITLE
will prisma support SurrealDB ?

### DISCUSSION THREAD

**Discussion by rickyraz** [prisma/prisma#15453] [Q&A] [answered]
its a fresh DB architecture btw..

**Comment by nurul3101** [comment, marked as answer]
Hi @rickyraz 👋 

It's not on the immediate roadmap yet. We do have a Feature Request for it: #15346
If you can leave a 👍 to the request, it would help our product team prioritise it.

**Reply by mysticaltech** [reply]
@nurul3101 It's gotten +121 👍 as of today. Please, maybe time to prioritize support for it! 🙏

**Reply by nurul3101** [reply]
@mysticaltech Decision regarding supporting a new connector is taken by our product and engineering team. Though this is not on our immediate [roadmap](https://prismaio.notion.site/Prisma-Roadmap-50766227b779464ab98899accb98295f), I'll share this internally to get the team's thoughts.

**Reply by mysticaltech** [reply]
Thank you!

**Reply by Ekwuno** [reply]
@nurul3101 it's up to +170 👍🏾 as of today and SurrealDB has 19.8K stars right now and a lot of users. I think this would be a great addition for Prisma users. As a Prisma user myself, i'd love to be able to work with SurrealDB

**Reply by nurul3101** [reply]
Thanks for the follow-up, It's great to see that SurrealDB has gained a lot of traction, with over 19.8K stars and a growing user base. I'll    highlight this to our team 🙏

**Reply by Keliqq** [reply]
I support you, I also love prisma very much and would like to support SurrealDB

**Reply by mysticaltech** [reply]
Please don't forget about this, SurrealDB is now one of the best DBs and has become widely adopted.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
